src/components/Tooltip/Tooltip.tsx

```diff
@@ -44,19 +44,34 @@ const Tooltip = ({
   const [inlineStyles, setInlineStyles] = useState({})
   const [inlineArrowStyles, setInlineArrowStyles] = useState({})
   const [show, setShow] = useState(false)
+  const [rendered, setRendered] = useState(false)
```

Comment:
added a new prop to handle if the tooltip can be rendered or not, this way we can handle the animations and the element render